docs/howto/configurations-ref.rst

```diff
@@ -0,0 +1,947 @@
+
+Core
```

Comment:
```suggestion

.. _config-ref/core:

``[core]``
``````````
Can you add labels? This will allow us to refer to fragments more easily. We should use precise section names case-sensitive.